### ISSUE TITLE
Shows file uploaded, even if there is no connection

### ISSUE THREAD

**Issue by stsdc** [Carglglz/upydev#17]
```shell
OSError: [Errno 113] No route to host
File Uploaded!
```

Should throw an error.

**Comment by Carglglz** [comment]
Are you sure you are using last version from  master repo?, ```upydev: 0.3.2```
I cannot reproduce this.

With laptop wifi off:
```
upydev put -f test_file.txt
Uploading file test_file.txt...
op:put, host:192.168.1.53, port:8266, passwd:********.
test_file.txt -> /test_file.txt
Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/3.7/bin/upytool", line 385, in <module>
    main()
  File "/Library/Frameworks/Python.framework/Versions/3.7/bin/upytool", line 363, in main
    s.connect(addr)
OSError: [Errno 51] Network is unreachable
```
With wifi on and device off:
It hangs for a while and then timeouts.
```
upydev put -f test_file.txt
Uploading file test_file.txt...
op:put, host:192.168.1.53, port:8266, passwd:********.
test_file.txt -> /test_file.txt
Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/3.7/bin/upytool", line 385, in <module>
    main()
  File "/Library/Frameworks/Python.framework/Versions/3.7/bin/upytool", line 363, in main
    s.connect(addr)
TimeoutError: [Errno 60] Operation timed out
```
With WebREPL connected and trying the command in another terminal window just hangs and  the WebREPL shows this:
```
Concurrent WebREPL connection from ('192.168.1.67', 55865) rejected
```

The code which was fixed in commit [ b973fdbcf](https://github.com/Carglglz/upydev/commit/b973fdbcfa544ed3b3ac2cb4201979be203f8c80) is this section

```
file_uploaded = subprocess.call(shlex.split(copyfile_str))
    if file_uploaded == 0:
        print('File Uploaded!')
```
This should prevent the message if the return code is not zero (which should happen if there is any error)

So it will be nice if you can write the steps to reproduce the error 👍 .

**Comment by stsdc** [comment]
My bad! pip version used. sorry 😬
